cardano-chain-gen/cardano-chain-gen.cabal

```diff
@@ -130,27 +130,6 @@ test-suite cardano-chain-gen
   other-modules:        Test.Cardano.Db.Mock.Config
                         Test.Cardano.Db.Mock.Examples
                         Test.Cardano.Db.Mock.Property.Property
-                        Test.Cardano.Db.Mock.Unit.Alonzo
-                        Test.Cardano.Db.Mock.Unit.Alonzo.Config
-                        Test.Cardano.Db.Mock.Unit.Alonzo.Plutus
-                        Test.Cardano.Db.Mock.Unit.Alonzo.PoolAndSmash
-                        Test.Cardano.Db.Mock.Unit.Alonzo.Reward
-                        Test.Cardano.Db.Mock.Unit.Alonzo.Simple
-                        Test.Cardano.Db.Mock.Unit.Alonzo.Stake
-                        Test.Cardano.Db.Mock.Unit.Alonzo.Tx
-                        Test.Cardano.Db.Mock.Unit.Babbage
-                        Test.Cardano.Db.Mock.Unit.Babbage.CommandLineArg.ConfigFile
-                        Test.Cardano.Db.Mock.Unit.Babbage.CommandLineArg.EpochDisabled
-                        Test.Cardano.Db.Mock.Unit.Babbage.Config.MigrateConsumedPruneTxOut
-                        Test.Cardano.Db.Mock.Unit.Babbage.Config.Parse
-                        Test.Cardano.Db.Mock.Unit.Babbage.InlineAndReference
-                        Test.Cardano.Db.Mock.Unit.Babbage.Other
-                        Test.Cardano.Db.Mock.Unit.Babbage.Plutus
-                        Test.Cardano.Db.Mock.Unit.Babbage.Reward
-                        Test.Cardano.Db.Mock.Unit.Babbage.Rollback
-                        Test.Cardano.Db.Mock.Unit.Babbage.Simple
-                        Test.Cardano.Db.Mock.Unit.Babbage.Stake
-                        Test.Cardano.Db.Mock.Unit.Babbage.Tx
```

Comment:
Are we sure we want to remove ALL past era tests?

Reply:
We could keep some tests as a reference and some that showcase differences in the behaviour between eras. We could filter them together in a meeting.